mycity/mycity/test/integration_tests/test_open_spaces_intent.py

```diff
@@ -1,21 +1,24 @@
 import csv
-import unittest
-import unittest.mock as mock
-import mycity.test.integration_tests.intent_test_mixins as mix_ins
-import mycity.test.integration_tests.intent_base_case as base_case
-import mycity.test.test_constants as test_constants
+from unittest import mock, skip
+
+from mycity.test import test_constants
+from mycity.test.integration_tests.intent_base_case import IntentBaseCase
+from mycity.test.integration_tests.intent_test_mixins import (
+    CardTitleTestMixIn,
+    CorrectSpeechOutputTestMixIn,
+    RepromptTextTestMixIn,
+)
 
 
 #############################################
 # TestCase class for get_open_spaces_intent #
 #############################################
 
-@unittest.skip('development on open spaces intent stalled!')
-class OpenSpacesTestCase(mix_ins.RepromptTextTestMixIn,
-                         mix_ins.CardTitleTestMixIn,
-                         mix_ins.CorrectSpeechOutputTestMixIn,
-                         base_case.IntentBaseCase):
-
+@skip('development on open spaces intent stalled!')
```

Comment:
I think we should remove this test since development on open spaces has been stalled for months now

Reply:
This was from a migration conflict I think? Not scoping it to this PR because this is meant to be exactly 0 functional changes.